src/components/category-card/category-card.stories.tsx

```diff
@@ -1,18 +1,18 @@
 import React from 'react';
-import { CategoryCardOne } from './category-card-one';
+import CategoryCard from './category-card';
 import type { Meta, StoryObj } from '@storybook/react';
-import { ActionFigure } from '@/assets/icons';
+import { IconActionFigure } from '@/assets/icons';
 
 export default {
 	title: 'Components/Category Card One',
```

Comment:
If you want to rename the component, make sure you remember to rename the story. Please update this title accordingly